### PR TITLE
My first site

### PR DESCRIPTION
**Pull request by ArQaer**

### CODE REVIEW

File: myFirstWebsite/css/styles.css [hunk 1]
```diff
@@ -96,55 +190,121 @@ body {
     font-weight: 500;
     font-size: 18px;
     line-height: 24px;
-    color: #FFFFFF;
+    color: var(--search-form-text-color);
     margin-right: 60px;
     padding-left: 43px;
     padding-bottom: 22px;
+    /* https://stackoverflow.com/questions/917610/put-icon-inside-input-element-in-a-form */
```
Comment (nastiaymlnv) [inline myFirstWebsite/css/styles.css]: It will be nice if you remove unnecessary code

File: myFirstWebsite/css/styles.css [hunk 2]
```diff
@@ -158,33 +318,484 @@ body {
...
+.locations__item-small {
+    grid-column: span 2;
+}
+
+.locations__item-image {
+    width: 100%;
+    object-fit: cover;
+}
+
+.locations__item-big > img {
```
Comment (nastiaymlnv) [inline myFirstWebsite/css/styles.css]: Get rid of nesting selectors. Look in BEM how to do it

File: myFirstWebsite/index.html [hunk 3]
```diff
@@ -118,13 +112,334 @@ <h2 class="benefits__item-title">
...
+                    <img src="./images/site-logo.svg" alt="site logo" class="footer__logo-image">
+                </div>
+                <nav class="footer__nav">
+                    <ul class="footer__list">
+                        <li class="footer__item">
+                            <h3 class="footer__item-title">
+                                Company
+                            </h3>
+                        </li>
+                        <li class="header__item">
```
Comment (nastiaymlnv) [inline myFirstWebsite/index.html]: Strange class name (header item in footer section). It's hard to change code in future. Check BEM

File: myFirstWebsite/index.html [hunk 4]
```diff
@@ -118,13 +112,334 @@ <h2 class="benefits__item-title">
...
+                        <img src="../myFirstWebsite/images/Sapporo.jpg" alt="Sapporo" class="locations__item-image"> 
+                        <h3 class="locations__item-title">
+                            Sapporo
+                        </h3>
+                    </div>
+                </div>
+            </div>
+            <div class="locations__offers-wrapper">
+                    <div class="locations__offer first-offer">
+                        <div class="offer__image first-offer-image">
```
Comment (andriiivaniuk) [inline myFirstWebsite/index.html]: I didn't find the implementation of **first-offer-image** class in your css code.

File: myFirstWebsite/index.html [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over. Being gorgeous with belly side up i could pee on this.
+                        </p>
+                        <button type="button" class="top-places__more-btn common-btn">see more</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="videos" id="videos">
+            <div class="videos__wrapper">
+                <div class="videos__adaptive-wrapper">
```
Comment (andriiivaniuk) [inline myFirstWebsite/index.html]: Very nice idea! Honestly I didn't find a proper solution to scale up a smaller video, but you managed it nicely! However, you don't need 2 wrappers here, you can have 1 like this: 


```
.videos__adaptive-wrapper{

    max-width: 1200px;
    margin: 0 auto;
    position: relative;
    padding-top: 30px;
    aspect-ratio: 16 / 9;
    overflow: hidden;
} 
```

**Comment by nastiaymlnv** [comment]
image
So much text in description

**Comment by nastiaymlnv** [comment]
image
Please put some interaction in images (scaling, opacity, or something else)

**Comment by nastiaymlnv** [comment]
image
Something went wrong with footer. Check it

**Comment by nastiaymlnv** [comment]
image
Add a little much space to these items (maybe put in a column)

**Comment by nastiaymlnv** [comment]
image
In this section two cards cause the page to scroll horizontally. Сheck this

File: myFirstWebsite/css/styles.css (hunk 1 above)
Comment (ArQaer) [reply myFirstWebsite/css/styles.css]: +

File: myFirstWebsite/index.html (hunk 3 above)
Comment (ArQaer) [reply myFirstWebsite/index.html]: +

File: myFirstWebsite/index.html (hunk 4 above)
Comment (ArQaer) [reply myFirstWebsite/index.html]: +

File: myFirstWebsite/index.html (hunk 5 above)
Comment (ArQaer) [reply myFirstWebsite/index.html]: +

File: myFirstWebsite/css/styles.css (hunk 2 above)
Comment (ArQaer) [reply myFirstWebsite/css/styles.css]: +

**Comment by andriiivaniuk** [comment]
Снимок1
This gap is too big on larger screens for some reason.

**Comment by andriiivaniuk** [comment]
Снимок2
I don't think that those lines should be so long after elements wrapping.

**Comment by andriiivaniuk** [comment]
Снимок333
You didn't use font-weight here, please check the design.

**Review by andriiivaniuk — APPROVED** [review]
good work :+1:

**Review by nastiaymlnv — APPROVED** [review]
You are doing a good job!

File: myFirstWebsite/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,826 @@
...
+    padding: 0 10px;
+}
+
+
+.search-form__field {
+    position: relative;
+    width: 100%;
+    max-width: 297px;
+    background: none;
+    background: url(../images/search-field-icon.svg) 0px 7px ;
```
Comment (VladyslavHybalo) [inline myFirstWebsite/css/styles.css:186]: Could you pls make this circle as pseudo-element?

File: myFirstWebsite/index.html [hunk 7]
```diff
@@ -0,0 +1,440 @@
...
+                </div>
+            </div>
+        </section>
+        <section class="locations" id="locations">
+            <div class="locations__wrapper ">
+                <div class="locations__title-wrapper">
+                    <h2 class="locations__title common-title custom-title-underline" >
+                        Get inspired for your next trip
+                    </h2>
+                    <a href="" class="locations__view-all-link common-link">
```
Comment (VladyslavHybalo) [inline myFirstWebsite/index.html:126]: locations__view-all-link looks like you hasn't declared `locations__view-all-link` into CSS file, pls get rid of all unused code.

File: myFirstWebsite/index.html [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+    <link rel="stylesheet" href="./css/fonts.css">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="shortcut icon" href="./images/site-icon-h179.png" type="image/png">
+</head>
+<body>
+    <div class="header__image-holder">
+        <header class="header">
+            <div class="wrapper">
+                <div class="header__wrapper">
+                    <div class="header__logo">
```
Comment (VladyslavHybalo) [inline myFirstWebsite/index.html:18]: It's a good practice to make a logo as a link to the main page, for SPA it usually scrolls to the top of the page

File: myFirstWebsite/index.html [hunk 9]
```diff
@@ -0,0 +1,440 @@
...
+                        <img src="./images/site-logo.svg" alt="site logo" class="header__logo-image">
```
Comment (VladyslavHybalo) [inline myFirstWebsite/index.html:19]: `image` here is an element, not modificator

File: myFirstWebsite/index.html [hunk 10]
```diff
@@ -0,0 +1,440 @@
...
+                        <div class="benefits__item-icon">
+                            <img src="./images/person-icon.png" alt="person">
+                        </div>
+                        <h2 class="benefits__item-title">
+                            Your Personal Japan Guide
+                        </h2>
+                        <p class="benefits__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over.
+                        </p>
+                        <a href="#!" class="benefits__item-more-link common-link">
```
Comment (VladyslavHybalo) [inline myFirstWebsite/index.html:99]: Pls check BEM naming conventions, looks like you have missed some there ideas

File: myFirstWebsite/css/styles.css [hunk 11]
```diff
@@ -0,0 +1,826 @@
...
+/* Locations ^^/
+
+/* Top-palaces */
+
+.top-places__wrapper {
+    max-width: 1262px;
+    margin: 0 auto;
+    padding: 0 10px;
+    margin-bottom: 86px;
+    margin-top: 139px;
```
Comment (VladyslavHybalo) [inline myFirstWebsite/css/styles.css:465]: nitpick: could be done by one margin property

File: myFirstWebsite/css/styles.css [hunk 12]
```diff
@@ -0,0 +1,826 @@
...
+.videos__adaptive-wrapper{
+    max-width: 1440px;
+    margin: 0 auto;
+    position: relative;
+    padding-top: 30px;
+    aspect-ratio: 16 / 9;
+    overflow: hidden;
+} 
+
+.videos__adaptive-wrapper video {
```
Comment (VladyslavHybalo) [inline myFirstWebsite/css/styles.css:582]: No need for such nesting

File: myFirstWebsite/css/styles.css [hunk 13]
```diff
@@ -0,0 +1,826 @@
...
+@media screen and (max-width: 940px) {
+    /* .search-form__field {
+        margin-bottom: 20px;
+        margin-right: 20px;
+    }
+
+    .search-form__submit-btn {
+        margin-bottom: 20px;
+    } */
+}
```
Comment (VladyslavHybalo) [inline myFirstWebsite/css/styles.css:699]: to remove

**Pull request merged**